kie-server-parent/kie-server-services/kie-server-services-kafka/src/main/java/org/kie/server/services/jbpm/kafka/RawJsonEventWriter.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2021 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+package org.kie.server.services.jbpm.kafka;
+
+import java.io.IOException;
+
+import org.kie.api.runtime.process.ProcessInstance;
+
+import static org.kie.server.services.jbpm.kafka.KafkaJsonUtils.jsonMarshallerWriter;
+
+
+public class RawJsonEventWriter implements KafkaEventWriter {
+
+    @Override
+    public byte[] writeEvent(ProcessInstance processInstance, Object value) throws IOException {
+        return jsonMarshallerWriter.marshallAsBytes(value);
```

Comment:
during the building of the write you can instantiate the json marshaller if you like with the right class loader. Don't use of statics.

Reply:
RawJsonEventWriter will use ClassLoader associated to the thread in which is executed, This is intentional and will work, I do not get the issue

Reply:
I changes the static marshaller  instance shared by RawJsonWriter and CloudEventWriter to private fields.